language/tools/vm-genesis/src/main.rs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) The Libra Core Contributors
```

Comment:
This is deleted in master; please don't bring it back!

Reply:
You have done great things, Sam, great things!